CONTRIBUTING.md

```diff
@@ -175,10 +175,12 @@ Project maintainers are responsible for releasing new versions of pycytominer.
 Creating a new release includes the following steps:
 
 1. Create a new branch from `main` for the release (e.g. `release-v1.0.0`)
-2. Run the command `cz bump --files-only` to update the version number in `CITATION.cff` and `pyproject.toml:tool.commitizen`.
-3. Review the changes to CHANGELOG.md and edit as needed.
-4. Review the [commit history](https://github.com/cytomining/pycytominer/compare) from the last release and add documentation for changes that weren't auto-included because they didn't follow the conventional-commit standard.
-5. `git add` all changes and run `cz bump` to create the release commit.
+2. Review the [commit history](https://github.com/cytomining/pycytominer/compare) from the last release and check whether it includes commits that don't follow the conventional commit standard.
+   If all changes follow conventional commits, skip to step 5.
+3. Run the command `cz bump --files-only` to update the version number in `CITATION.cff` and `pyproject.toml:tool.commitizen` and generate the draft changelog.
+4. Review the changes to CHANGELOG.md and add descriptions of changes that didn't use conventional commit.
```

Comment:
Minor question (maybe overly semantic, sorry!): would we ever need to also modify a description (not just add/append to existing content)? If so, consider changing "... add descriptions ..." to "... modify commit descriptions to meet conventional commit specifications." (or something like this). It could also be important to see a link here with concerns to where these specifications are commonly referenced for maintainers.